.github/workflows/node.js.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
         strategy:
             matrix:
-                node-version: [14.x, 16.14]
+                node-version: [14.x, 16.x, 18.x]
```

Comment:
Also, fixes `16.14` which doesn't make sense.

Reply:
hmm could it be due to this Fetch API?

```bash
(node:113293) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
```

Reply:
It seems the msw is not properly supporting node v.18 yet. 
As per this open issue: https://github.com/mswjs/msw/issues/1388

Reply:
This issue is fixed in #967... closing this PR